relayer/src/link/relay_path.rs

```diff
@@ -1343,34 +1364,32 @@ impl<ChainA: ChainHandle, ChainB: ChainHandle> RelayPath<ChainA, ChainB> {
                     IbcEvent::SendPacket(e) => {
                         // Catch any SendPacket event that timed-out
                         if self.send_packet_event_handled(e)? {
-                            debug!(
-                                "[{}] refreshing schedule: already handled send packet {}",
-                                self, e
-                            );
+                            debug!("[{}] already handled send packet {}", self, e);
                         } else if let Some(new_msg) =
                             self.build_timeout_from_send_packet_event(e, &dst_status)?
                         {
-                            debug!(
-                                "[{}] refreshing schedule: found a timed-out msg in the op data {}",
-                                self, odata
-                            );
-                            timed_out.entry(odata_pos).or_insert_with(Vec::new).push(
-                                TransitMessage {
+                            debug!("[{}] found a timed-out msg in the op data {}", self, odata);
```

Comment:
The id of `odata` could be put into KV here, too, rather than formatting in-message.

Reply:
I think the `Display` impl of odata provides more context, and it's usually useful to have those additional details (the target, the height, and the batch size).

Reply:
OK, it's just some logging backends allow you to do filtering by e.g. the `tracking_id` key, but baking it in would have the filter miss that.

Reply:
Used `odata` in 85c60fa8f1b37e0f823aaa3cf03c79bafd8ef994.